package Javacore.O_Exception.CheckedExceptions;

import javax.security.auth.login.LoginException;

public class Funcionario extends Pessoa {
    
    public void salvar(){
        
    }
    
    public void salvar2() throws LoginException{
        
    }
    
}
